dev/generate_bom.py

```diff
@@ -202,9 +202,13 @@ def write_bom(self):
     output_yaml[VERSION] = toplevel_with_build
     self.__bom_file = '{0}.yml'.format(toplevel_with_build)
     self.write_bom_file(self.__bom_file, output_yaml)
-    self.publish_bom(self.__bom_file)
     output_yaml[VERSION] = 'nightly'
```

Comment:
"nightly" is the bleeding edge, unstable set of microservices we build nightly. I added a flag to enable building and publishing this BOM, default to `False`.